Formula/curl.rb

```diff
@@ -67,6 +77,21 @@ def install
     args << (build.with?("gssapi") ? "--with-gssapi" : "--without-gssapi")
     args << (build.with?("rtmpdump") ? "--with-librtmp" : "--without-librtmp")
 
+    if build.with? "libidn2"
+      resource("libidn2").stage do
+        system "./configure", "--disable-dependency-tracking",
+                              "--disable-silent-rules",
+                              "--prefix=#{libexec}/vendor/libidn2",
```

Comment:
GitHub didn't tell me about half these comments via email, what is going on 😓. I know `curl` at least can be _really_ fussy when it comes to using static libraries, but I'll check it out.